### DISCUSSION TITLE
Change working Width

### DISCUSSION THREAD

**Discussion by mzaliac14** [scfmod/FS22_TerraFarm#70] [General]
Hello!! First of all hats off to the amazing mod you've created. 

I'd like to know if the working width of the terrafarm mod can be changed on the vehicles. I'm using the FSMiner's Komatsu grader which has working width much wider than the actual blade. Is there anyway to change that or any hotkey to shrink the worknig width? Much appreciated for your respone.

**Comment by scfmod** [comment]
You can edit Komatsu_GD655.xml in xml_configurations/fsm
There's a list of nodes in terraform.i3d, by changing the nodes you can alter the width :) I.e remove the first and last it will go from 3.8 to 2.32 width. Or you can tighten the x values of them all etc.

**Reply by mzaliac14** [reply]
Thank you.
